### ISSUE TITLE
BUG - [High] - [All] - Accessibility 5103 - Voiceover and Talkback don't acknowledge the error after submitting 5103 waiver without required fields, stays on the button

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#12216]
## What happened?

When submitting a 5103 waiver without the checkbox checked, both iOS (Voiceover) and Android (Talkback) just remain on the submit waiver button without saying anything about there being an error.  Non-sighted user has no idea what the problem may be.

## Specs:

- Device: iPhone, Android
- OS: iOS, Android
- User Account: +228  (Cory Sohrakoff can set claims to need 5103 flow, if needed.)
- Accessibility State: Screen reader ON

Slack thread for adding user data:  https://dsva.slack.com/archives/C09B6NPTBC4

## Steps to Reproduce

1. Take steps to submit a 5103 waiver with screen reader ON.
2. Submit without checking the required checkbox
3. Both iOS and Android remain on the submit waiver button without acknowledging the error that occured.

## Desired behavior

Error should be acknowledged - screen reader should jump up to error.

## Acceptance Criteria

Error is acknowledged - screen reader jumps to error and informs user there is an error.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High
- Frequency:  Low 

- 2 - High

## Linked to Story

Story: https://github.com/department-of-veterans-affairs/va-mobile-app/issues/11992

PR:  https://github.com/department-of-veterans-affairs/va-mobile-app/pull/12205

## Screen shot(s) and additional information

<img width="664" height="766" alt="Image" src="https://github.com/user-attachments/assets/e01bae2f-f435-479e-a1f3-e658ce908c52" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
